bitnami/pubsub/kafka-shared/kustomization.yaml

```diff
@@ -6,7 +6,36 @@ resources:
   - ../../shared/logging
   - certs.yaml
 
+components:
+  - ../../../shared/opslevel
+
 patches:
   - path: kafka-shared-admins.yaml
   - path: refresh-broker-certs.yaml
   - path: refresh-exporter-certs.yaml
+  - target:
+      version: v1
+      group: apps
+      kind: Deployment
+      name: kafka-shared-exporter
+      namespace: pubsub
+    patch: |-
+      - op: add
+        path: /metadata/annotations/app.uw.systems~1description
+        value: Metrics exporter for shared Kafka
+      - op: add
+        path: /metadata/annotations/app.uw.systems~1tier
+        value: tier_4
+  - target:
+      version: v1
+      group: apps
+      kind: StatefulSet
+      name: kafka-shared
+      namespace: pubsub
+    patch: |-
+      - op: add
+        path: /metadata/annotations/app.uw.systems~1description
+        value: Kafka instance for the shared cluster
+      - op: add
+        path: /metadata/annotations/app.uw.systems~1tier
+        value: tier_4
```

Comment:
I think kafka instances have much higher tier (1?) by default

Reply:
> I think kafka instances have much higher tier (1?) by default

Good point, :facepalm: I meant to open a discussion about that when I raised the PR. Tier 1 seems reasonable though, since I believe we support any service consuming from this

Reply:
I'm inclining also to tier_1 as if kafka breaks most likely services won't work. But it depends on what these services do.. as kafka is not exposed directly to customers.
I think we could default to tier 1 and if teams need lower they can overwrite.

For mirror-maker tier 4 is fine